src/Cli/Microsoft.DotNet.Cli.Sln.Internal/Microsoft.DotNet.Cli.Sln.Internal.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyName>Microsoft.DotNet.Cli.Sln.Internal</AssemblyName>
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
     <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PublicSign Condition="!$([MSBuild]::IsOSPlatform('Windows'))">true</PublicSign>
```

Comment:
Is there a more readable way to check for non-Windows? I think the ! at the beginning is really easy to miss because it is next to so many other punctuation symbols. =“!$([

It kind of gets lost in my eyes

Reply:
At the moment, `IsWindows` and `IsUnixLike` are exclusive. But I'm hesitant to make `!IsWindows` calls into `IsUnixLike` because there's a theoretical future distinction there.

Reply:
We could use `Condition="$([MSBuild]::IsOSPlatform('Windows')) == false"` to make it more obvious.

Reply:
@akoeplinger great idea, love it. Did it.